cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -531,12 +538,6 @@ policy "aws_ri_recommendations" do
       field "instanceClass" do
         label "Instance Class"
       end
-      field "instanceSize" do
```

Comment:
Shouldn't it be renamed to size?

Reply:
I was told that instance type, instance size and node type should be mapped to resource type. Also, that depending on the resource type, only one of them would be populated, so that's why that information is inside "resource type".
You can check line 375 to see how that's concatenated.

Reply:
I don't think it's right to delete all the fields by taking one of them as a resource type. Reports can be viewed manually as well, or do all fields have the same value?

Reply:
Need a consultation @TDubovchenko

Reply:
I've added those fields again. I think it has more sence to keep them.